angular-workspace/projects/ni/nimble-angular/src/directives/table-column/text/tests/nimble-table-column-text.directive.spec.ts

```diff
@@ -0,0 +1,111 @@
+import { Component, ElementRef, ViewChild } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { NimbleTableModule } from '../../../table/nimble-table.module';
+import { NimbleTableColumnTextModule } from '../nimble-table-column-text.module';
+import { NimbleTableColumnTextDirective, TableColumnText } from '../nimble-table-column-text.directive';
+
+describe('NimbleTableColumnText', () => {
+    describe('module', () => {
+        beforeEach(() => {
+            TestBed.configureTestingModule({
+                imports: [NimbleTableModule]
+            });
+        });
+
+        it('custom element is defined', () => {
+            expect(customElements.get('nimble-table')).not.toBeUndefined();
+        });
+    });
+
+    describe('with template string values', () => {
+        @Component({
+            template: `
+                <nimble-table>
+                    <nimble-table-column-text #column field-name="field1" placeholder="no value"></nimble-table-column-text>
+                </nimble-table>
+            `
+        })
+        class TestHostComponent {
+            @ViewChild('column', { read: NimbleTableColumnTextDirective }) public directive: NimbleTableColumnTextDirective;
+            @ViewChild('column', { read: ElementRef }) public elementRef: ElementRef<TableColumnText>;
+        }
+
+        let fixture: ComponentFixture<TestHostComponent>;
+        let directive: NimbleTableColumnTextDirective;
+        let nativeElement: TableColumnText;
+        beforeEach(() => {
+            TestBed.configureTestingModule({
+                declarations: [TestHostComponent],
+                imports: [NimbleTableColumnTextModule, NimbleTableModule]
+            });
+
+            fixture = TestBed.createComponent(TestHostComponent);
+            fixture.detectChanges();
+            directive = fixture.componentInstance.directive;
+            nativeElement = fixture.componentInstance.elementRef.nativeElement;
+        });
+
+        it('will use template string values for fieldName', () => {
+            expect(directive.fieldName).toBe('field1');
+            expect(nativeElement.fieldName).toBe('field1');
+        });
+
+        it('cwill use template string values for placeholder', () => {
+            expect(directive.placeholder).toBe('no value');
+            expect(nativeElement.placeholder).toBe('no value');
+        });
+    });
+
+    describe('with property bound values', () => {
```

Comment:
Most of our Angular tests also have a `describe('with attribute bound values')` that tests that bindings to `attr.foo` work correctly. Is there a reason you omitted that section of tests for this directive?

Reply:
Just missed it. Thanks.